deployments/utoronto/image/Dockerfile

```diff
@@ -91,6 +91,23 @@ RUN apt-get update -qq --yes && \
         libxss1 \
         > /dev/null
 
+# for kableExtra & partitions & prettydoc & RSelenium & scholar & showtext & stopwords
+# See https://github.com/utoronto-2i2c/jupyterhub-deploy/issues/25
+RUN apt-get update -qq --yes && \
+    apt-get install --yes -qq \
+        libcurl4-openssl-dev \
+        libxml2-dev \
+        imagemagick \
+        libmagick++-dev \
+        libgmp3-dev \
+        libicu-dev \
+        libssl-dev \
+        libglpk-dev \
+        libfreetype6-dev \
+        libgit2-dev \
+        git \
```

Comment:
Awesome! \o/

I *think* git is already included in the base image, and doesn't need to be installed again. I *think* that might also be true for libssl and libcurl4-openssl-dev, but am not sure. Can you verify that all these packages aren't already installed?

Thank you for making this happen!

Reply:
Thanks @yuvipanda <3 I revisited the list and removed the pkgs already installed.